examples/3d/deferred_rendering.rs

```diff
@@ -129,7 +129,7 @@ fn setup(
         ..default()
     });
 
-    let sphere_color = Color::rgb(10.0, 4.0, 1.0);
+    let sphere_color = Color::rgb(1.0, 0.4, 0.1);
```

Comment:
This was intentionally very bright to test RGB9E5 usage.

Reply:
forgot i changed this, reverted